packages/gatsby-theme-learning/src/components/config-context.ts

```diff
@@ -1,19 +1,24 @@
 import { createContext } from 'react';
 
+export enum EFeatureFlag {
+  CourseStatus = 'status-indicator',
```

Comment:
I wonder if a data structure that  has defaults might be better suited.  I can't really wrap my head around why exactly to be honest, it's just a feeling.

Reply:
not sure what you mean here. but the `feature flag` feature is working and I would descope this discussion from this PR. We can have a discussion about it and then create a separate ticket